src/main/java/org/takes/rs/RsVelocity.java

```diff
@@ -189,4 +205,31 @@ public Pair(final CharSequence key, final Object obj) {
             super(key, obj);
         }
     }
+
+    /**
+     * Template folder.
+     */
+    public static final class TemplateFolder {
```

Comment:
You are right the TemplateFolder is miss concept, I just need the path to the folder and this can be represented by String, the initially idea was to have some kind of validation to check if it's a really folder, but this is just an velocity configuration option.
I fix the code and push again.
Thanks for the review.
Cheers!